debian/bionic/foreman-proxy/foreman-proxy.install

```diff
@@ -0,0 +1,22 @@
+foreman-proxy                             etc
+bin                                       usr/share/foreman-proxy
+bundler.d/bmc.rb                          usr/share/foreman-proxy/bundler.d
+bundler.d/dhcp_isc.rb                     usr/share/foreman-proxy/bundler.d
+bundler.d/krb5.rb                         usr/share/foreman-proxy/bundler.d
+bundler.d/libvirt.rb                      usr/share/foreman-proxy/bundler.d
+bundler.d/puppetca_token_whitelisting.rb  usr/share/foreman-proxy/bundler.d
+bundler.d/puppet.rb                       usr/share/foreman-proxy/bundler.d
```

Comment:
I wonder if we should drop this. It's only used if Puppet < 3 is used and there are no packages for it on Bionic.

Reply:
I guess there might be people out there you built funky special setups to stay on the old version, so I'd rather leave this as is for now.